tests/acceptance/smoke-test.js

```diff
@@ -1,97 +1,64 @@
-import Ember from 'ember';
-import { module, test, ok } from 'qunit';
-import startApp from '../helpers/start-app';
+import { visit, fillIn } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import { module, test } from 'qunit';
 
-let application;
+module('Acceptance: Smoke', function (hooks) {
```

Comment:
The tests were copied from #82, is that correct?

Reply:
Yes